src/test/kotlin/day11/DefaultFileOperationsTest.kt

```
@@ -0,0 +1,107 @@
+import java.io.File
+import java.io.PrintStream
+
+
+class DefaultFileOperationsTest {
+    private val defaultFileOperations = DefaultFileOperations()
+    private val sourcePaths = mutableListOf("origin1.txt", "origin2.txt", "origin3.txt")
+    private val targetPaths = mutableListOf("copy1.txt", "copy2.txt", "copy3.txt")
+
+    @Before
+    fun setUp() {
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)